src/Test/EventListenerIntrospectionTrait.php

```diff
@@ -9,7 +9,7 @@
 
 namespace Zend\EventManager\Test;
 
-use PHPUnit_Framework_Assert as Assert;
+use PHPUnit\Framework\Assert;
```

Comment:
why?

in that case we are in a trait class and the class that this trait import can not have that method.